wqflask/wqflask/templates/links.html

```diff
@@ -3,19 +3,22 @@
 {% block title %}Links{% endblock %}
 
 {% block css %}
-<!-- <link rel="stylesheet" type="text/css" href="/static/new/css/markdown.css" /> -->
+<link rel="stylesheet" type="text/css" href="/static/new/css/markdown.css" />
 {% endblock %}
 
 {% block content %}
 
- <div class="github-btn">
- <a href="https://github.com/genenetwork/gn-docs">Edit on Github</a>	
- </div>
+<div class="github-btn-container">
+    <div class="github-btn ">
+        <a href="https://github.com/genenetwork/gn-docs">
+            Edit on Github inner
```

Comment:
**suggestion:** @Alexanderlacuna Could you make this text read: "Edit Text" and let that apply to all other pages?